README.md

```diff
@@ -1,12 +1,11 @@
-# News Site
+# Project-news-site - Technigo week 2 - buil a responsive webpages
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Technigo Remote Frontend Boot Camp. Our task was to create a classic magazine or newspaper site that should contain a header with a name of the site, a logo and a navbar. The site should also change depending on if you visit the site from a mobile, tablet or desktop. Further the site should have a big news section and a grid of cards with other news. On desktop sized screens, there should be four cards in a row, two in a row on tablets and on mobile, there should just be one card on each row.
+## Work 
+The sketch was made in Canva and then  the different modules were interpreted in code. 
+The site is built on html and css.
+## Result 
+The site is very basic and the mobile view is the best way to look at it. 
```

Comment:
It would be great to know more about why you think the mobile view is the best. How would you have liked the tablet and desktop to display differently?